wordpress-mobile/AztecEditor-Android
# Issue/370 add toolbar toggle

### Fix
Add advanced toolbar mode as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/370.

### Test
0. Add `aztec:advanced="true"` to [AztecToolbar](https://github.com/wordpress-mobile/AztecEditor-Android/blob/develop/app/src/main/res/layout-v17/activity_main.xml#L8-L13) view.
1. Launch demonstration app.
2. Notice toolbar contains the following format buttons:
- Media
- Headings
- Lists
- Quote
- Bold
- Italic
3. Tap horizontal ellipsis button.
4. Notice toolbar expands to contain the following format buttons:
- Media
- Headings
- Lists
- Quote
- Bold
- Italic
- Link
- Underline
- Strikethrough
- Horizontal Rule
- More
- HTML
5. Tap vertical ellipsis button.
6. Notice toolbar collapses to contain the original format buttons.
